repos/system_upgrade/el7toel8/actors/checkosrelease/libraries/library.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from leapp.exceptions import StopActorExecution, StopActorExecutionError
+from leapp.libraries.common import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import OSReleaseFacts
+
+
+def skip_check():
+    """ Check if has environment variable to skip this actor checks """
+    if os.getenv('LEAPP_SKIP_CHECK_OS_RELEASE'):
+        reporting.report_generic(
+            title='Skipped OS release check',
+            severity='low',
```

Comment:
I can't help myself but I have to disagree - I feel like the severity of this is quite high - It might be the reason for other hidden failures.

Reply:
Yeah, I'd also make it medium or high.